lib/chef/resource/windows_firewall_rule.rb

```diff
@@ -39,6 +39,19 @@ class WindowsFirewallRule < Chef::Resource
       end
       ```
 
+      **Configuring multiple remote-address ports on a rule**:
+
+      ```ruby
+      windows_firewall_rule 'MyRule' do
+        description          "testing out remote address arrays"
+        enabled              false
+        local_port           1434
+        remote_address       %w(10.17.3.101 172.7.7.53)
+        protocol             "TCP"
```

Comment:
```suggestion
        protocol             'TCP'
```

Reply:
Isn't that going to blow up and tell me double-quotes are preferred?